### PR TITLE
Fixed typo

### LINKED ISSUE #778: :open_book: `Docs`: Spelling errors in core-concept.md
### What should be changed?

> # Core Concepts Overview

Let's assume you're familiar with core Git, Docker, Kubernetes, Continuous `Integratoin`, ...

`Integratoin` should be changed to `Integration`

### PR DESCRIPTION
**Pull request by PrajwalBorkar**
Signed-off-by: Prajwal Borkar <prajwalborkar5075@gmail.com>

## Pre-Checklist

Note: please complete **_ALL_** items in the following checklist.

- [x] I have read through the [CONTRIBUTING.md](https://github.com/devstream-io/devstream/blob/main/CONTRIBUTING.md) documentation.
- [ ] My code has the necessary comments and documentation (if needed).
- [ ] I have added relevant tests

## Description

Fixed typo in `core-concepts.md` file
## Related Issues

Fixes #778

### CODE REVIEW

**Comment by aFlyBird0** [comment]
Thank you for your second pr!  ✊

DevStream community follows the [conventional commits specification](https://www.conventionalcommits.org/en/v1.0.0/). Please read our [doc](https://docs.devstream.io/en/latest/development/commit-messages/) .

The right commit message could be : `docs: fix typo`.

You can choose to update this commit message and use `git push -f` to overide old commits. Or, do nothing and let us change commit messages when we merge it.

**Review by aFlyBird0 — APPROVED** [review]

**Comment by aFlyBird0** [comment]
Could you please start a review? @hxcGit  Thanks a lot! (please pay attention to commit messages)

**Review by hxcGit — APPROVED** [review]

**Comment by aFlyBird0** [comment]
The "Code Owner" of this part is IronCore. He must approve pr before it can be merged, please be patient. @PrajwalBorkar 
image

**Review by IronCore864 — APPROVED** [review]

**Pull request merged**
